compiler/z/codegen/OMRCodeGenerator.cpp

```diff
@@ -6095,16 +6095,6 @@ OMR::Z::CodeGenerator::supportsMergingOfHCRGuards()
           !self()->comp()->compileRelocatableCode();
    }
 
-bool
-OMR::Z::CodeGenerator::isConcurrentScavengeEnabled()
-   {
-#if defined(OMR_GC_CONCURRENT_SCAVENGER)
-   return TR::Compiler->target.cpu.getS390SupportsGuardedStorageFacility();
-#else
-   return false;
-#endif
-   }
```

Comment:
I'm not sure you can delete this, unless every single project that builds on top of OMR implements this function - for example there's code in `omr/compiler/z/codegen/ControlFlowEvaulator.cpp` that does a `if (cg->isConcurrentScavengeEnabled())` query which would fail to resolve at runtime. I think you would need to keep it in and have it return false (probably just in the .hpp file)